### PR TITLE
Fix up the commits page

### PR DESCRIPTION
**Pull request by FangedTerror802**
It was a little messed up with the centers... Anyways, I personally think it looks much better now.

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing :+1:`:+1:`, or :-1:`:-1:`.
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both :+1:up- and :-1:down-votes are disregarded.
Comments containing monkeys are disregarded. :monkey: :monkey_face: :see_no_evil: :hear_no_evil: :speak_no_evil:
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:*
 3b1a67a55d4128627ef249d5b5660d2ecdb667f3

File: data/views/statistics.hbs [hunk 1]
```diff
@@ -23,4 +23,4 @@
             <td>{{os.hostname}}</td>
         </tr>
     </table>
-</center>
\ No newline at end of file
+</center>
```
Comment (shalecraig) [inline data/views/statistics.hbs]: :+1: that

**Comment by shalecraig** [comment]
:+1:

**Comment by samis** [comment]
:+1:

**Comment by OLef1air** [comment]
:+1:

**Comment by AlphaT3ch** [comment]
:+1:

**Comment by thenightwassave** [comment]
:+1:

**Comment by botwillacceptanything** [comment]
#### &lt;a target=&quot;_blank&quot; href=&quot;http:&#x2F;&#x2F;thecatapi.com&#x2F;?id=b5&quot;&gt;&lt;img src=&quot;http:&#x2F;&#x2F;24.media.tumblr.com&#x2F;ZabOTt2mpdpa8utjEJQndAAi_400.jpg&quot;&gt;&lt;&#x2F;a&gt;
:+1: The vote passed! This PR will now be merged into master.


----
**Tallies:**
:+1:: 6 (100%)
:-1:: 0 (0%)

These following 1 voter(s) were not stargazers, so their votes were not counted:

- :monkey: @coveralls

Be sure to :star:star the repository if you want to have your say!

**Pull request merged**